qcodes_contrib_drivers/drivers/AimTTi/EL320P.py

```diff
@@ -0,0 +1,81 @@
+from typing import List, Dict, Union, Optional, Sequence, Any, Tuple
+import numpy as np
+import visa
+import time
+
+from qcodes.instrument.visa import VisaInstrument
+import qcodes.utils.validators as vals
+
+class EL320P(VisaInstrument):
+    """Qcodes driver for AIM & Thurlby Thandar EL320P power supply (BlueFors 4K warmup heater).
+    """
+    def __init__(self, name, address, **kwargs):
+        super().__init__(name, address, terminator='\r\n', **kwargs)
+        self.visa_handle.baud_rate = 9600
+        self.add_parameter(
+            name='voltage_set',
+            label='Voltage setting',
+            unit='V',
+            get_cmd='V?',
+            get_parser=self._get_parser,
+            set_cmd='V {}',
+            vals=vals.Numbers(0,30)
+            )
+        self.add_parameter(
+            name='voltage_out',
+            label='Actual output voltage',
+            unit='V',
+            get_cmd='VO?',
+            get_parser=self._get_parser
+            )
+        self.add_parameter(
+            name='current_set',
+            label='Current setting',
+            unit='A',
+            get_cmd='I?',
+            get_parser=self._get_parser,
+            set_cmd='I {}',
+            vals=vals.Numbers(0.01,2)
+            )
+        self.add_parameter(
+            name='current_out',
+            label='Actual output current',
+            unit='A',
+            get_cmd='IO?',
+            get_parser=self._get_parser
+            )
+        self.add_parameter(
+            name='mode',
+            label='Mode',
+            unit='',
```

Comment:
If there is no applicable units, let us just drop `unit`? There are other parameters with `unit` is defined as empty string, my comment applies also to them.